### PR TITLE
Buzzer Sample for android things with pwm.

### PR DESCRIPTION
**Pull request by vintummala**
Signed-off-by: Vineela Tummalapalli <vineela.tummalapalli@intel.com>

### CODE REVIEW

**Comment by fujunwei** [comment]
The buzzer will buzz after flashing new [DP4 early image](http://bjbeare-svr.ostc.intel.com/junk/halton-iot_joule-flashfiles-eng.bjbeare.zip)  in  j12 pin 5 without install any application manually, and it has been buzzing. Does it means the image has supported this feature? or i used incorrect pin.

**Comment by bjbeare** [comment]
Not entirely, but we are close.

File: build.gradle [hunk 1]
```diff
@@ -29,9 +29,9 @@ buildscript {
 allprojects {
     repositories {
         // Uncomment this to use the test-version of UPM and MRAA
-        // maven {
-        //    url "http://dl.bintray.com/android-ia/UPM-Test"
-        // }
+         maven {
```
Comment (bjbeare) [inline build.gradle]: please re-comment this out and re-submit pull request.

File: settings.gradle [hunk 2]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-include ':accel_on_lcd'
+include ':accel_on_lcd', ':grovebuzzer'
```
Comment (bjbeare) [inline settings.gradle]: add line break.

File: grovebuzzer/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<resources>
+    <string name="app_name">grovebuzzer</string>
+
+    <string name="Buzzer_Edison_Arduino">PWM0</string>
+    <string name="Buzzer_Edison_Sparkfun">GP20</string>
```
Comment (bjbeare) [inline grovebuzzer/src/main/res/values/strings.xml]: should be pwm0 for sparkfun as well.

**Pull request merged**